docs/index.md

```diff
@@ -15,1621 +15,118 @@ parent="smn_workw_machine"
 
-Once your Docker host has been created, it then has a number of commands for
-managing them:
+Once you create one or more Docker hosts, Docker Machine supplies a number of commands for
+managing them. Using these commands you can 
 
- - Starting, stopping, restarting
- - Upgrading Docker
- - Configuring the Docker client to talk to your host
+ - start, inspect, stop, and restart a host
+ - upgrade the Docker client and daemon
+ - configure a Docker client to talk to your host
 
-## Getting help
-
-Docker Machine is still in its infancy and under active development. If you need
-help, would like to contribute, or simply want to talk about to the project with
-like-minded individuals, we have a number of open channels for communication.
-
-- To report bugs or file feature requests: please use the [issue tracker on
-  Github](https://github.com/docker/machine/issues).
-- To talk about the project with people in real time: please join the
-  `#docker-machine` channel on IRC.
-- To contribute code or documentation changes: please [submit a pull request on
-  Github](https://github.com/docker/machine/pulls).
-
-For more information and resources, please visit
-[https://docs.docker.com/project/get-help/](https://docs.docker.com/project/get-help/).
 
-## Installation
+## A simple Docker Machine workflow
 
-Docker Machine is supported on Windows, OS X, and Linux and is installable as one
-standalone binary. The links to the binaries for the various platforms and
-architectures are below:
+Once you install Docker Machine, you can create and manage machines. A very
+simple workflow on a local system has three steps.
 
-- [Windows - 32bit](https://github.com/docker/machine/releases/download/v0.3.0/docker-machine_windows-386.exe)
-- [Windows - 64bit](https://github.com/docker/machine/releases/download/v0.3.0/docker-machine_windows-amd64.exe)
-- [OSX - x86_64](https://github.com/docker/machine/releases/download/v0.3.0/docker-machine_darwin-amd64)
-- [OSX - (old macs)](https://github.com/docker/machine/releases/download/v0.3.0/docker-machine_darwin-386)
-- [Linux - x86_64](https://github.com/docker/machine/releases/download/v0.3.0/docker-machine_linux-amd64)
-- [Linux - i386](https://github.com/docker/machine/releases/download/v0.3.0/docker-machine_linux-386)
-
-### OS X and Linux
-
-To install on OS X or Linux, download the proper binary to somewhere in your
-`PATH` (e.g. `/usr/local/bin`) and make it executable. For instance, to install on
-most OS X machines these commands should suffice:
-
-```
-$ curl -L https://github.com/docker/machine/releases/download/v0.3.0/docker-machine_darwin-amd64 > /usr/local/bin/docker-machine
-$ chmod +x /usr/local/bin/docker-machine
-```
-
-For Linux, just substitute "linux" for "darwin" in the binary name above.
-
-Now you should be able to check the version with `docker-machine -v`:
+The first step is to create a new machine.  For example, to create a machine
+called `my-docker-machine` on your local system.
```

Comment:
This is the name used throughout the example. 1) It is easier to see than `dev` and 2) the name serves as a mnemonic for both the product and the thing the user is learning to create.